apps/keyboard/js/imes/jshangul/jshangul.js

```diff
@@ -34,7 +32,7 @@
 
     var UTF8_HANGUL_SYLLABLES_START = 0xAC00;
     var UTF8_HANGUL_COMPATIBILITY_START = 0x3131;
-    var UTF8_JUNGSEONG_START = 0x1161;
+    // var UTF8_JUNGSEONG_START = 0x1161;
```

Comment:
I think we could find this line with `git log -S`  command, so please just remove the dead code.